astropy/utils/data.py

```diff
@@ -871,18 +871,31 @@ def compute_hash(localfn):
     return h.hexdigest()
 
 
-def _find_pkg_data_path(data_name, package=None):
-    """
-    Look for data in the source-included data directories and return the
-    path.
-    """
+def get_pkg_data_path(*path, package=None):
```

Comment:
```suggestion
def get_pkg_data_path(path, package=None):
```

Reply:
I think making this take `*args` would be useful though?  Are there any examples out there of code using this function without explicitly using `package=` as a keyword argument?  That's the only thing this would break, and if it does break that's not our fault since it was a private-ish function to begin with.